src/main/java/org/apache/commons/validator/Validator.java

```diff
@@ -196,7 +196,11 @@ public Validator(final ValidatorResources resources, final String formName, fina
     public void clear() {
         formName = null;
         fieldName = null;
-        parameters = new HashMap<>();
+        if(parameters == null){
```

Comment:
How can `parameters` ever be null?

Reply:
- https://github.com/apache/commons-validator/commit/2d983f2f2a5e5523754ff88c611250204e2dbf66#r150814815